### PR TITLE
Stealth Balancing for 0.5 (part 1)

### PR DESCRIPTION
**Pull request by agrrr3**
conservative stealth revamp for 0.5; first part, uncritical and base changes (stringtable + adding NamedInteger support to python parser)

please review/merge ASAP (i will go with whatever stringtable suggestions you want). merging sooner saves me rebasing troubles.

### CODE REVIEW

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -6487,7 +6487,7 @@ METER_DETECTION_VALUE_DESC
 
 Space ships and planets have a Detection Range meter. There is the option to display Detection Range circles (in uu) in the Galaxy Map section of the Options menu. To facilitate predictions of what would be within a Detection Range, there are also options to display a standard 'Map Scale Line' as well as a 'Map Scale Circle' centered on the currently selected system and indicating the same distance as the Map Scale Line. They are both responsive to the the map zoom level and can be toggled via hotkeys.
 
-Some objects or abilities may allow vision of certain areas outside of this range. Some examples are from species abilities of [[species SP_TRITH]] or [[species SP_GEORGE]], or from the effects from a [[buildingtype BLD_SCRYING_SPHERE]].
+Some objects or abilities may allow vision of certain areas outside of this range. Some examples are species abilities of [[species SP_FULVER]], [[species SP_GEORGE]], [[SP_NIGHTSIDERS]] or [[species SP_TRITH]], or the effects from a [[buildingtype BLD_SCRYING_SPHERE]], or finding a map in a derelict scout when exploring new systems.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]:
```suggestion
Some content may allow vision outside of this range. This includes the species abilities of [[species SP_FULVER]], [[species SP_GEORGE]], [[SP_NIGHTSIDERS]] or [[species SP_TRITH]], the [[buildingtype BLD_SCRYING_SPHERE]], or finding a map in a derelict scout when exploring new systems.
```

**Review by o01eg — APPROVED** [review]
Looks fine

**Comment by agrrr3** [comment]
merge, please

**Comment by agrrr3** [comment]
@geoffthemedio @o01eg dont have the rights to merge myself; i merged the review output of geoff; the NamedInteger works as expected

**Pull request merged**